gettingstarted/beforestarting.rst

```diff
@@ -12,6 +12,18 @@ This section of the guide will teach you basic information about terminology use
    This is a lot of info to take in, but if you are a complete noob to the Switch scene, you should seriously consider reading all of it. The Switch scene is very different from the 3DS/WiiU/Wii.
 
 ........
+

```

Comment:
I'm going to say don't bother talking too much about NAND backups (or any actual process at all) this early. It can be confusing to talk about using hack tools before you've even started talking about what the tools are. It almost sounds like this is supposed to be something you do from OFW before you start hacking.

It should be fine to leave this in Safety Precautions. I'll beef up the warning on that page to make it clear that you should not skip it.